consul/metadata.csv

```diff
@@ -1,154 +1,145 @@
+consul.raft.state.candidate,count,10,event,,[DogStatsD] [Prometheus]The number of initiated leader elections,-1,consul,leader elecs init
+consul.raft.apply,count,10,transaction,,[DogStatsD] [Prometheus] The number of raft transactions occurring,0,consul,raft trans
+consul.raft.commitTime.avg,gauge,10,millisecond,,[DogStatsD] [Prometheus] The average time it takes to commit a new entry to the raft log on the leader,-1,consul,raft cmt avg
+consul.raft.commitTime.count,count,10,,,[DogStatsD] [Prometheus] The number of samples of raft.commitTime,0,consul,raft cmt cnt
+consul.raft.commitTime.max,gauge,10,millisecond,,[DogStatsD] [Prometheus] The max time it takes to commit a new entry to the raft log on the leader,-1,consul,raft cmt max
+consul.raft.commitTime.median,gauge,10,millisecond,,[DogStatsD] [Prometheus] The median time it takes to commit a new entry to the raft log on the leader,-1,consul,raft cmt med
+consul.raft.commitTime.95percentile,gauge,10,millisecond,,[DogStatsD] [Prometheus] The p95 time it takes to commit a new entry to the raft log on the leader,-1,consul,raft cmt p95
+consul.raft.commitTime.quantile,gauge,,millisecond,,[DogStatsD] [Prometheus] The quantile time it takes to commit a new entry to the raft log on the leader,-1,consul,raft cmt q
+consul.raft.commitTime.sum,count,,millisecond,,[DogStatsD] [Prometheus] The sum of the time it takes to commit a new entry to the raft log on the leader,-1,consul,raft cmt sum
+consul.raft.leader.dispatchLog.avg,gauge,10,millisecond,,[DogStatsD] [Prometheus] The average time it takes for the leader to write log entries to disk,-1,consul,dispatch log avg
+consul.raft.leader.dispatchLog.count,count,10,,,[DogStatsD] [Prometheus] The number of samples of raft.leader.dispatchLog,0,consul,dispatch log cnt
+consul.raft.leader.dispatchLog.max,gauge,10,millisecond,,[DogStatsD] [Prometheus] The max time it takes for the leader to write log entries to disk,-1,consul,dispatch log max
+consul.raft.leader.dispatchLog.median,gauge,10,millisecond,,[DogStatsD] [Prometheus] The median time it takes for the leader to write log entries to disk,-1,consul,dispatch log med
+consul.raft.leader.dispatchLog.95percentile,gauge,10,millisecond,,[DogStatsD] [Prometheus] The p95 time it takes for the leader to write log entries to disk,-1,consul,dispatch log p95
+consul.raft.leader.dispatchLog.quantile,gauge,,millisecond,,[DogStatsD] [Prometheus] The quantile time it takes for the leader to write log entries to disk,-1,consul,dispatch log q
+consul.raft.leader.dispatchLog.sum,count,,millisecond,,[DogStatsD] [Prometheus] The sum of the time it takes for the leader to write log entries to disk,-1,consul,dispatch log sum
+consul.raft.leader.lastContact.avg,gauge,10,millisecond,,[DogStatsD] [Prometheus] Average time elapsed since the leader was last able to check its lease with followers,-1,consul,last contact avg
```

Comment:
```suggestion
consul.raft.leader.lastContact.avg,gauge,10,millisecond,,[DogStatsD] [Prometheus] The average time elapsed since the leader was last able to check its lease with followers,-1,consul,last contact avg
```